starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -36,7 +37,18 @@ public void setup() {
   public void tearDown() {
     this.mongoTemplate.dropCollection(Inventory.class);
   }
-
+  // Test the create function. Initialize inventory and then add it to mongo collection with the create function. Print the ID before and after to check for null
+  @Test
+  public void createTest() {
+    Inventory inventory1 = new Inventory();
+    inventory1.setName(NAME);
+    inventory1.setId(ID);
+    inventory1.setProductType(PRODUCT_TYPE);
+    System.out.print("The initial ID is: " + inventory1.getId());
+    this.inventoryDAO.create(inventory1);
+    System.out.println("After Create Method ID: " + inventory1.getId());
+    Assert.assertTrue(inventory1.getId() != ID);
```

Comment:
Can you also add asserts for verifying that the name and product type were saved properly.